ocaml/libs/tracing/tracing.ml

```diff
@@ -634,3 +634,34 @@ let with_tracing ?(attributes = []) ?(parent = None) ~name f =
     | Error e ->
         warn "Failed to start tracing: %s" (Printexc.to_string e) ;
         f None
+
+module EnvHelpers = struct
+  let traceparent_key = "TRACEPARENT"
+
+  let of_traceparent traceparent =
+    match traceparent with
+    | None ->
+        []
+    | Some traceparent ->
+        [String.concat "=" [traceparent_key; traceparent]]
+
+  let to_traceparent env =
+    let env_opt =
+      List.find_opt (String.starts_with ~prefix:traceparent_key) env
+    in
+    Option.bind env_opt (fun key_value ->
+        match String.split_on_char '=' key_value with
+        | [key; traceparent] when String.equal key traceparent_key ->
+            Some traceparent
+        | _ ->
+            None
+    )
```

Comment:
I agree! I did it this way because env vars where already pass in this format for the smapi observer. 

Plus, this is the rpc representation for env vars in this scenario. When I attempted to implement the server side, I still needed to split by `=`. So it made sense to transform it back and forth in the same way.